integration/src/main/resources/servers/drshub-local.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "drshub-local",
+  "description": "drshub Local env.",
+
+  "samUri": "https://sam.dsde-dev.broadinstitute.org",
+  "policyManagerUri": "http://localhost:8080",
```

Comment:
Actually, this `policyManagerUri` value is supposed to be specific to ECM - and each service is supposed to have its own URI field in the TestRunner configs. It's a bit silly in my opinion, and there's been a lot of back and forth in the #testrunner slack channel about changing that approach. For now we probably need to make a quick change in the the test runner codebase (similar to [this](https://github.com/DataBiosphere/terra-test-runner/pull/29/files)) to give drshub its own URI config value.

Reply:
If we make a PR for that on the testrunner repo, it may also be worthwhile to rename the ECM config to a more clear name than "policyManagerURI".